### ISSUE TITLE
Tooltip examples should be made keyboard accessible

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#3961]
**Describe the issue. What is the expected and unexpected behavior?**
When an experience can be initiated by mouse, a similar experience should be provided to keyboard and assistive technology users. Currently the first two tooltip examples are not keyboard accessible.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
Tab through the tooltips documentation, the first two examples do not show the tooltip because they are text and text does not normally receive focus. These spans should have a tabIndex in order to allow them to receive focus.

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
This is an example bug.

**Comment by jessiehuff** [comment]
It looks like this change got accidentally reverted in one of our merges into master from release: https://github.com/patternfly/patternfly-react/pull/4123/files?file-filters%5B%5D=.md#diff-dbcf37813f42d8c8ac6f747fe5aad7bcR40
